### DISCUSSION TITLE
Post server action error

### DISCUSSION THREAD

**Discussion by Hanmant1** [vercel/next.js#73170] [Help]
### Summary

If I make POST request from POSTMAN with invalid request body to page which invoke server action during the initial load.

Getting below error response with 500 status code.

0:["$@1",["U-nhTjoGnQNE1xixWxKjs",null]]
1:E{"digest":"874560660"}

When you say digest, are you signing the error message with a hashing algorithm?
 
What about this, 0:["$@1",["**U-nhTjoGnQNE1xixWxKjs**",null]]

Which serialization mechanism next.JS is using for this? How we conclude what is mean by **U-nhTjoGnQNE1xixWxKjs**?

### Additional information

```js
When I tried reproduce in the dev Mode then **U-nhTjoGnQNE1xixWxKjs** replaced by server mode e.g., development/test/production.

0:["$@1",["development",null]]
1:E{"digest":"2301226142","message":"...."}
```

**Comment by Hanmant1** [comment]
Hello all, I would appreciate it if you could check and respond as soon as possible. And let me know if you need any input from me.
Tim Neutkens ([@timneutkens](https://twitter.com/timneutkens))
Naoyuki Kanezawa ([@nkzawa](https://twitter.com/nkzawa))
Guillermo Rauch ([@rauchg](https://twitter.com/rauchg))
Arunoda Susiripala ([@arunoda](https://twitter.com/arunoda))
Tony Kovanen ([@tonykovanen](https://twitter.com/tonykovanen))
Dan Zajdband ([@impronunciable](https://twitter.com/impronunciable))

**Comment by icyJoseph** [comment]
What are you trying to do here?

Server Actions execute code on the server, and the mechanism used, are POST requests, against the current pathname, something like `fetch('', { method: 'POST', /* other stuff */}`.

So the question is, what is it you are trying to achieve?
